listenbrainz/webserver/views/user.py

```diff
@@ -141,6 +149,7 @@ def profile(user_name):
         "profile_url": url_for('user.profile', user_name=user_name),
         "mode": "listens",
         "spotify": spotify_data,
+        "youtube": youtube_data,
```

Comment:
There are other pages where we need to add the youtube data to the props:

webserver/views/player.py
webserver/views/playlist.py
webserver/views/recommendations_cf_recording.py

(I know this is a WIP but just noting them down before I forget…)
Those global app props do sound useful right about now ! :p